addons/api/mirage/generated/factories/host-set.js

```diff
@@ -5,11 +5,20 @@ import { random, date, datatype } from 'faker';
  * GeneratedHostSetModelFactory
  */
 export default Factory.extend({
-  type: 'static',
+  type: () => random.arrayElement(['static', 'plugin']),
```

Comment:
@randallmorey I think here we want the same `type` and `plugin.name` than the host-catalog it pertains?

Reply:
Correct.

Reply:
Gotcha, I updated the PR following recommendations.